R/new_ode_model.R

```diff
@@ -293,8 +297,27 @@ new_ode_model <- function (model = NULL,
       if(!file.exists(new_folder)) {
         dir.create(new_folder)
       }
-      file.copy(from = templ_folder, to = new_folder,
-                overwrite = TRUE, recursive = TRUE, copy.mode = FALSE)
+      file.copy(
+        from = templ_folder,
+        to = new_folder,
+        overwrite = TRUE,
+        recursive = TRUE,
+        copy.mode = FALSE
+      )
+
+      ## copy definition JSON, if model created from definition file.
+      if(!is.null(definition) && file.exists(definition)) {
+        dir.create(file.path(new_folder, "inst"))
+        copy_result <- file.copy(
+          from = definition,
+          to = file.path(new_folder, "inst", "definition.json")
```

Comment:
```suggestion
          to = file.path(new_folder, "inst", "definition.json5")
```

Reply:
Yup, if we just copy like this, it should be `.json5` since there are multiline strings that regular JSON-parsers can't handle. I'd prefer to keep `definition` though to have the same name in each package.